pkg/providers/pricing/zz_generated.pricing.go

```diff
@@ -88,8 +88,6 @@ func init() {
 		// c7gn family
 		"c7gn.12xlarge": 2.995200, "c7gn.16xlarge": 3.993600, "c7gn.2xlarge": 0.499200, "c7gn.4xlarge": 0.998400,
 		"c7gn.8xlarge": 1.996800, "c7gn.large": 0.124800, "c7gn.medium": 0.062400, "c7gn.xlarge": 0.249600,
-		// cc2 family
-		"cc2.8xlarge": 2.000000,
```

Comment:
Any clue why these were removed? Is there record of these instance types being taken out?

Reply:
@engedaam Can you take a look at this and see if these instance types are removed or if there was something that caused these prices to get removed?

Reply:
Both those instance types have been deprecated